tools/developer.html

```diff
@@ -108,37 +219,56 @@ <h3>Markup</h3>
         editor.find('.rendering_stats').text(rendering_stats);
         editor.find('.rendering_time').text(rendering_time);
         editor.find('output').text(result.value);
-        SourceStore.save(source, language);
+        saveSettings();
       });
 
       $('.editor button#show-structure').click(function(e) {
         var editor = $(this).parents('.editor');
         editor.toggleClass('visible-structure');
+
+        localStorage.setItem(key_STRUCTURE,
+          editor.hasClass('visible-structure'));
```

Comment:
```suggestion
        localStorage.setItem(key_STRUCTURE, editor.hasClass('visible-structure'));
```

not sure why this was on two lines

Reply:
Not sure it mattered, but ok. :)